PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -228,6 +228,7 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         pfRelIso03_chg_quadratic = Var("userFloat('PFIsoChgQuadratic')/pt",float,doc="PF relative isolation dR=0.3, charged hadron component (with quadraticEA*rho*rho + linearEA*rho Winter22V1 corrections)"),
         pfRelIso03_all_quadratic = Var("userFloat('PFIsoAllQuadratic')/pt",float,doc="PF relative isolation dR=0.3, total (with quadraticEA*rho*rho + linearEA*rho Winter22V1 corrections)"),
         hoe = Var("hadronicOverEm()",float,doc="H over E",precision=8),
+        hoe_TowOverEm = Var("hadTowOverEm()",float,doc="H over E Tower based calculation",precision=8),
```

Comment:
@TizianoBevilacqua I found the name `hoe_TowOverEm` a bit confusing as `oe`:=`OverEm`. Would it be better to just call it `hadTowOverEm` or `hoe_Tower`?